.github/workflows/build.yml

```diff
@@ -0,0 +1,145 @@
+name: Test & Build
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test:
+    name: Test Firm Deputy Hub
```

Comment:
In the infra repo, this is named `Test Deputy Hub` which is why an extra step is appearing in the build below. I will change it to `Test Firm Deputy Hub` in the infra repo in the next ticket